lib/getComment.js

```diff
@@ -1,17 +1,13 @@
-module.exports = getComment
-
 function getComment (comment, defaultComment) {
-  // comment can be either a string or an array
-
-  if (!comment) {
-    comment = defaultComment
-  }
+  // set comment to default if none passed it
+  const eventComment = (!comment) ? defaultComment : comment
```

Comment:
for future reference, I'd prefer style changes like these (basically all of `lib/getComment.js`) in a separate PR.

Reply:
oh sorry, would you prefer i pull it out and make it a separate PR?